### PR TITLE
Ml example to multi band

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[EuroCrops-Beispiel auf Multi-Band-Raster anpassen](https://trello.com/c/EoLTrClj/1571-eurocrops-beispiel-auf-multi-band-raster-anpassen)

File: geoengine/raster.py [hunk 1]
```diff
@@ -212,3 +212,93 @@ def from_ge_record_batch(record_batch: pa.RecordBatch) -> RasterTile2D:
...
+        return stack
+
+
+async def tile_stream_to_np_stack(raster_stream: AsyncIterator[RasterTile2D]) -> AsyncIterator[RasterTileStack2D]:
+
+    ''' Convert a stream of raster tiles to stream of stacked tiles '''
+    store: List[RasterTile2D] = []
+    first_band: int = -1
+
+    async for tile in raster_stream:
```
Comment (michaelmattig) [inline geoengine/raster.py:275]: would it make sense to do some sanity checks here that the correct tiles arrived? Or should be trust the backend?

File: geoengine/raster.py [hunk 2]
```diff
@@ -212,3 +212,93 @@ def from_ge_record_batch(record_batch: pa.RecordBatch) -> RasterTile2D:
...
+        return stack
+
+    def to_xarray(self, clip_with_bounds: Optional[gety.SpatialBounds] = None) -> xr.DataArray:
+        '''Return the raster stack as an xarray.DataArray'''
+        arrays = [self.single_band(i).to_xarray(clip_with_bounds) for i in range(0, len(self.data))]
+        stack = xr.concat(arrays, dim='band')
+        return stack
+
+
+async def tile_stream_to_np_stack(raster_stream: AsyncIterator[RasterTile2D]) -> AsyncIterator[RasterTileStack2D]:
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply geoengine/raster.py]: [b106f4d](https://github.com/geo-engine/geoengine-python/pull/172/commits/b106f4d3e853ee3ccc77afef7d2d787c003c1626)

File: geoengine/raster.py (hunk 1 above)
Comment (jdroenner) [reply geoengine/raster.py:275]: [a7db320](https://github.com/geo-engine/geoengine-python/pull/172/commits/a7db3207e3997c866e28dfede7b439fe7b7ccb48)

then had to change more things:

[ad0bca8](https://github.com/geo-engine/geoengine-python/pull/172/commits/ad0bca8e154a75444fc57695e503b6f57b192120)

**Review by michaelmattig — DISMISSED** [review]

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**